config/locales/fr.yml

```diff
@@ -0,0 +1,25 @@
+fr:
+  refinery:
+    plugins:
+      refinerycms_copywriting:
+        title: Copywriting
+        title_tab: Copywriting
+    copywriting:
+      admin:
+        phrases:
+          locale_filters:
+            change_language: Changer la langue
+          scope_filters:
+            filter_by_scope: Filtrer par contexte
+            filtered_by: "Filtré par '%{filter_scope}'"
+            clear_scope_filter: Supprimer le filtre
+          records:
+            no_items_yet: Il n'y a pas de phrases de copywriting.
```

Comment:
Its a bit strange to leave `copywriting` there but I'm not sure what to do with that case neither

Maybe make it a "nom propre" and refer to the plugin?

```
Il n'y a pas de phrases dans Copywriting.
```

Or translate but there isn't an exact word for copywriting in french. I would need rewording for to make it work. i.e.; contenu, rédaction ?